salt/data-pipeline/nifi-registry.sls

```diff
@@ -4,7 +4,7 @@
 download-nifi-registry:
     file.managed:
         - name: /root/downloads/nifi-registry-0.4.0-bin.tar.gz
-        - source: https://www-eu.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
+        - source: http://archive.apache.org/dist/nifi/nifi-registry/nifi-registry-0.4.0/nifi-registry-0.4.0-bin.tar.gz
```

Comment:
I didn't pay attention when copying the URL, my mistake. Maybe better to keep using https.